### DISCUSSION TITLE
How to handle markers with Flutter?

### DISCUSSION THREAD

**Discussion by danielxavier89** [xvrh/lottie-flutter#192] [Q&A]
Hello! I have found great tutorials for web and swift, but nothing for Flutter. I'm not a developer, so I might have been overlooking something basic. 

Thanks!

**Comment by xvrh** [comment]
There is a small example in the example project here: https://github.com/xvrh/lottie-flutter/blob/master/example/lib/examples/markers.dart

I'm not aware of any tutorial unfortunately
